docs/DynamicESDL/DESDL-Mods/ESDLbooleanbool.xml

```diff
@@ -1,14 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
 "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
-<sect2 id="boolean_bool">
-  <title><emphasis role="bold">boolean bool</emphasis></title>
+<sect1 id="boolean_bool">
```

Comment:
please change "boolean_bool" to ESDL_boolean or ESDL_boolean_bool for consistency with other id attributes.